### PR TITLE
Tamil Language translation done.

### PR DESCRIPTION
**Pull request by manimaran96**
All strings are translated to Tamil.

### CODE REVIEW

**Comment by Sav22999** [comment]
To translate the app use Crowdin please, to avoid conflicts

File: app/src/main/res/values-ta/strings.xml [hunk 1]
```diff
@@ -32,169 +32,166 @@
...
+    <string name="daily_goal_is_not_official_by_mozilla">இந்த தினசரி இலக்கு அதிகாரப்பூர்வ Mozilla அம்சம் அல்ல. இது இந்த செயலியின் ஒரு அம்சம் மட்டுமே.</string>
     <!-- Message dialog (unspecified category) -->
-    <string name="message_dialog_new_version_available">A new version of the app is available (release {{*{{n_version}}*}}). You can update it from your favourite store (F-Droid, Google Play Store) or directly from GitHub.</string>
-    <!-- Others -->
-    <!-- DO NOT TRANSLATE OR CHANGE THIS - Languages (list) sort "A-Z" - "languages" and "languages_short" sort MUST coincide (e.g. Italian and it, English and en, ect.) -->
-    <!-- DO NOT TRANSLATE OR CHANGE THIS - Abbreviation of Common Voice (e.g. Italian -> it, Swedish -> sv-SE, ect.) -->
+    <string name="message_dialog_new_version_available">செயலியின் புதிய பதிப்பு(வெளியீடு : {{*{{n_version}}*}}) கிடைக்கிறது. நீங்கள் Playstore/F-Droid அல்லது நேரடியாக Github-ல் இருந்து புதுப்பிக்கலாம்.</string>
```
Comment (Sav22999) [inline app/src/main/res/values-ta/strings.xml:196]: Google Play Store is a registered trademark by Google, so do not change it (It's not "Playstore" -> "Google Play Store", better just as "Google Play").

Comment (manimaran96) [reply app/src/main/res/values-ta/strings.xml]: Oh. Ok I Will fix.

Comment (Sav22999) [reply app/src/main/res/values-ta/strings.xml]: @manimaran96 as I said, please use Crowdin.

**Comment by Sav22999** [comment]
@manimaran96 you can import easily these translations on Crowdin in this way:
image

**Comment by manimaran96** [comment]
@Sav22999 done. Review and update.

**Comment by Sav22999** [comment]
@manimaran96 thanks. I close this PR and I accept all translations on Crowdin 😄

**Pull request closed without merging**